particles/Processing/js/Dispose.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * @license
+ * Copyright (c) 2019 Google Inc. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * Code distributed by Google as part of this project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+defineParticle(({DomParticle, resolver, log}) => {
+
+  importScripts(resolver(`$here/tf.js`));
```

Comment:
Is it possible that we can inject tf? 

```
defineParticle(({DomParticle, resolver, log, tf}) => {
...
```

Reply:
Yes, it is possible. But right now tf.js belongs strictly to the particles and not to runtime itself, which is a desirable property.

Reply:
Cool. We can defer this to an issue for a future PR.

Reply:
Or, have a discussion about it over there.